test/jest/FullSave.test.ts

```diff
@@ -11,12 +11,14 @@ import { Companies } from "../../src/Company/Companies";
 describe("Check Save File Continuity", () => {
   establishInitialConditions();
   // Calling getSaveString forces save info to update
-  saveObject.getSaveData();
+  saveObject.getSaveData().catch((error) => {
```

Comment:
Since this is a test, we want this error to propagate. Also, presumably we want this sequenced. So, `await` send like the answer here, but the test will have to become async.

Reply:
I'll use `beforeAll`. Other solutions that do not work:
- async `describe`: `describe` does not accept an async function.
- If we put test code inside `then`, Jest does not recognize the tests.